src/web/app/pages-student/student-home-page/student-home-page.component.ts

```diff
@@ -114,7 +114,8 @@ export class StudentHomePageComponent implements OnInit {
             });
           });
 
-          this.courses.sort((a: StudentCourse, b: StudentCourse) => (a.course.courseId > b.course.courseId ? 1 : -1));
+          this.courses.sort((a: StudentCourse, b: StudentCourse) =>
+            (a.course.creationTimestamp < b.course.creationTimestamp ? 1 : -1));
           this.courses.slice(0, 3).forEach((course: StudentCourse) => {
             course.isTabExpanded = true;
             this.loadFeedbackSessionsForCourse(course.course.courseId);
```

Comment:
Would suggest replacing this with `this.sortCoursesBy(this.sortBy)`, the rationale being that there is already a function that does this functionality, and also this change makes it easier for others to see at a high level what is going on (i.e. that we are sorting courses by the current/default sort by)

Reply:
I have implemented the change, hope that's what you meant :)

Reply:
Yes that works, thanks!